whitelists/whitelist.yml

```diff
@@ -316,15 +330,9 @@
   address: 5Fjmh8QuPkLAcErd6oAGTACvy99ifEA5W3QeCnKVfUxH3uC6
 - name: address
```

Comment:
There are still a few addresses without a name. Is this correct?

Reply:
yes emma, i will send those tomorrow so the team can help me to identify them